theseus/theseus_layer.py

```diff
@@ -25,11 +25,16 @@
 
 
 class TheseusLayer(nn.Module):
-    def __init__(self, optimizer: Optimizer, vectorize: bool = True):
+    def __init__(
+        self,
+        optimizer: Optimizer,
+        vectorize: bool = True,
+        empty_cuda_cache: bool = False,
```

Comment:
Is it possible to ignore `empty_cuda_cache` for sparse or CPU solvers?

Reply:
I don't think this is necessary, since the default is `False`. Presumably this should only be turned on if the user thinks it will be useful.